src/localize/LocaleMacro.hx

```diff
@@ -9,6 +9,20 @@ class LocaleMacro {
         var fields = Context.getBuildFields();
         for (field in fields) {
             switch (field.kind) {
+                case FVar(type, _):
```

Comment:
Alright, I reverted those changes but added a opt-in compiler warning with the flag `hxlocalize_verbose`. So people can know what's going on, if they need to.